lerobot/common/envs/utils.py

```diff
@@ -86,3 +90,37 @@ def env_to_policy_features(env_cfg: EnvConfig) -> dict[str, PolicyFeature]:
         policy_features[policy_key] = feature
 
     return policy_features
+
+
+def check_all_envs_same_type(env: gym.vector.VectorEnv) -> bool:
```

Comment:
We tend to start with "are" and "is" in the repo for this use case
```suggestion
def are_all_envs_same_type(env: gym.vector.VectorEnv) -> bool:
```

Reply:
done